spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -233,11 +233,12 @@ def aggregate_per_slice_or_level(metric, mask=None, slices=[], levels=[], persli
     :return: Aggregated metric
     """
     if vert_level:
-        vert_level_slices = Image(vert_level).change_orientation('RPI').data.shape[2]
-        metric_slices = metric.data.shape[0]
+        vert_level_slices = Image(vert_level).change_orientation('RPI').data.shape[-1]
+        metric_slices = metric.data.shape[-1]
```

Comment:
Hrmm, wait a minute. Is `[-1]` what I want? 4D images have `t` as their last dimension, right?

Reply:
I've changed this to more explicitly index the I-S slices depending on the shape of the data.